<?xml version="1.0"?>

<!--
Launch file for Speech Recognition stack.
author: Nicolas Tobis
date: 05/19
-->

<launch>

    <!-- Launch speech recognition server -->
    <node pkg="speech_rec" type="google_speech_rec_service.py" name="speech_recognition"
      output="screen"/>


</launch>
